validationTemplate/src/ModelValidationTemplate/ValidationTemplate.py

```diff
@@ -0,0 +1,315 @@
+"""
+Created on August 2nd 2021
+@author: garrm
+
+This module is part of the RAVEN Validation template
+"""
+from __future__ import division, print_function, unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default', DeprecationWarning)
+# standard library
+import os
+import sys
+# external libraries
+# RAVEN libraries
+## caution: using abspath here does not do desirable things on Windows Mingw.
+import _utils as hutils
+raven_path = os.path.abspath(hutils.get_raven_loc()) 
+sys.path.append(raven_path)
+# Importing XML functions from RAVEN 
+from utils import xmlUtils
+# Importing Template base class from RAVEN
+from InputTemplates.TemplateBaseClass import Template
+
+
+
+class ValidationTemplate(Template):
+    """
+    Template class to create the validation file for the RAVEN validation template
+    """
+    #Template.addNamingTemplates({'step name': '{action}_{subject}', 'distribution': '{var}_dist', 'metric var': '{metric}_{var}'})
+
+    def __init__(self, metrics, distributions, calibration):
+        """ 
+        Validation Template Constructor 
+            @ In, metrics, dict, dictionary of metrics implemented with information
+            @ In, distributions, dict, dictionary of distributions implemented with information
+            @ In, calibration, boolean, indicates if the validation XML file will be used for calibration
+            @ Out, None 
+        """
+        super(ValidationTemplate, self).__init__()
+        self.metrics = metrics
+        self.distributions = distributions
+        self.calibration = calibration
+    
+    # =============================================================================
+    #     API methods
+    # =============================================================================
+    
+    def createWorkflow(self, model=None, exp=None, variables=None, metric=None, romTag=False, **kwargs):
+        """
+        Creates a new RAVEN workflow file based on the information in kwargs.
+        Specific to individual templates. Must overload to be useful.
+            @ In, romTag, bool, whether validation workflow should be written for 
```

Comment:
it looks like romTag has been moved to an optional entry. For optional entries, the note `optional` should appear, for example,
```
  @ In, romTag, bool, optional, whether ...
```
The order of the arguments should match the call order, as well.